src/app/downloads/downloads-chart.component.ts

```diff
@@ -112,12 +112,16 @@ export class DownloadsChartComponent implements OnDestroy {
   }
 
   dateFormat(): Function {
-    if (this.filter && this.filter.interval) {
-      switch (this.filter.interval.key) {
-        case INTERVAL_DAILY.key:
+    if (this.filter) {
+      switch (this.filter.interval) {
+        case INTERVAL_MONTHLY:
+          return monthYearFormat;
```

Comment:
maybe this should be formatted Oct '17 instead of 10/17? I imagine date formatting will come up either in internal review or user testing.